Keas.Mvc/Controllers/Api/WorkstationsController.cs

```diff
@@ -233,8 +233,22 @@ public async Task<IActionResult> Create([FromBody] Workstation workstation)
             {
                 return BadRequest();
             }
-            if (workstation.Space != null)
+
+            //Validate passed team matches workstation team.
+            var team = await _context.Teams.SingleAsync(a => a.Slug == Team);
```

Comment:
you only need to pull out the teamId here so you can `.Select()` and save a few bytes and cycles